src/Handlers.jl

```diff
@@ -141,6 +141,16 @@ function handle(r::Router, req)
     return handle(handler, req)
 end
 
-handle(hf::HandlerFunction, http::HTTP.Stream) = handle_request(hf.func,http)
+function handle(r::Router, http::HTTP.Stream)
+    request::HTTP.Request = http.message
+    request.body = read(http)
+    hf = gethandler(r,request)
+    request.response::HTTP.Response = hf.func(request)
+    startwrite(http)
+    write(http, request.response.body)
+    return
+end
+
+handle(hf::HandlerFunction, http::HTTP.Stream) = HTTP.Servers.handle_request(hf.func,http)
```

Comment:
This is the wrong dependency order; `Servers.jl` depends on `Handlers.jl`, not vice-versa. So we should define `handle_request` to be in `Handlers.jl` instead and have `Servers.jl` import/call it.

Reply:
Sounds good 👍 I can modify the PR accordingly.